examples/bri-3/src/bri/auth/guards/sri-local.guard.ts

```diff
@@ -0,0 +1,5 @@
+import { Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+
+@Injectable()
+export class SriLocalAuthGuard extends AuthGuard('local') {}
```

Comment:
why we need both jwt and local auth guards?

Reply:
Right, we just need jwt guard. Removing the other one